GetandTake/Areas/Identity/Pages/Account/ConfirmEmail.cshtml.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Text;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.AspNetCore.WebUtilities;
+
+namespace GetandTake.Areas.Identity.Pages.Account;
+
+public class ConfirmEmailModel : PageModel
+{
+    private readonly UserManager<IdentityUser> _userManager;
+
+    public ConfirmEmailModel(UserManager<IdentityUser> userManager)
+    {
+        _userManager = userManager;
+    }
+
+    [TempData]
+    public string StatusMessage { get; set; }
+    public async Task<IActionResult> OnGetAsync(string userId, string code)
```

Comment:
Know, this code is autogenerated, it should be refactored as well to make your codebase better and consistent. Let's use `string.IsNullOrWhiteSpace` (do it for the similar usages, please)